### DISCUSSION TITLE
Frequency To Band check

### DISCUSSION THREAD

**Discussion by karlg1** [wavelog/wavelog#1212] [Ideas]
It appears the frequency to band check is not working as it should and will allow a mismatch between the band and frequency during the live or post QSO entry. 
Is this a problem with my configuration or is the function not implemented?

Karl - KB2AS

**Comment by HB9HIL** [comment]
coming soon

**Reply by karlg1** [reply]
Thanks

**Reply by HB9HIL** [reply]
@karlg1  Was added with https://github.com/wavelog/wavelog/pull/1196 and is ready to test in `dev` branch. Will be in the next release :)

**Comment by karlg1** [comment]
Thanks for the update, works great.

Karl
